cmd/preflight/cli/root.go

```diff
@@ -16,7 +16,7 @@ var (
 
 func RootCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "preflight [url]",
+		Use:   "preflight [url-or-file]",
```

Comment:
https://github.com/replicatedhq/troubleshoot/blob/master/cmd/preflight/cli/root.go#L19

for consistency, let's just specify `url` in the help text

Reply:
Yeah, that's why I changed BOTH preflight and support-bundle in order to be consistent.  But I can go change them both back to just `url` if you want.